packages/data-lake-api/package.json

```diff
@@ -15,8 +15,11 @@
     "lint": "eslint . --ext .ts",
     "lint:all": "yarn run lint \"src/**/*.{ts,jsx}\"",
     "lint:fix": "yarn lint --fix",
-    "test": "jest --passWithNoTests",
-    "test:coverage": "yarn test --coverage"
+    "test": "yarn workspace @tupaia/data-lake-api check-test-data-lake-exists && DATA_LAKE_DB_NAME=data_lake_test jest --runInBand",
+    "test:coverage": "yarn test --coverage",
+    "update-test-data": "bash -c 'source .env && pg_dump -s -h data-lake-db.tupaia.org -U $DATA_LAKE_DB_USER -O $DATA_LAKE_DB_NAME -t analytics > src/__tests__/testData/testDataDump.sql'",
```

Comment:
Explicitly pull the schema off the `data-lake-db.tupaia.org`, rather than what's configured for `DATA_LAKE_DB_URL`. This is because we set `DATA_LAKE_DB_URL` to localhost for the tests, and we need a real database here to pull the schema. Should be fine to use as we're using a read-only user and it's just one table 👍

Reply:
Very handy!